db/migrate/20160805000209_create_contacts.rb

```diff
@@ -0,0 +1,17 @@
+class CreateContacts < ActiveRecord::Migration
+  def change
+    create_table :contacts do |t|
+      t.string :first_name
+      t.string :last_name
+      t.string :email
+      t.string :phone
+      t.string :service
+      t.date :date
+      t.string :availability
+      t.integer :skillset
+      t.text :notes
+
+      t.timestamps null: false
+    end
+  end
+end
```

Comment:
First name, Last Name, skillset, date,

Can we rename date to date_met and what is service? Should be more clear? Service or social thing I met them through?